.github/workflows/integration.yml

```diff
@@ -26,14 +26,10 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php-versions: ['7.4', '8.0', '8.1', '8.2']
+        php-versions: ['7.4', '8.2']
         databases: ['mysql', 'pgsql', 'sqlite']
-        server-versions: ['stable25', 'stable26', 'stable27', 'master']
```

Comment:
In case you still want to cover all of these (which makes sense if you want to support them), you can use an `include:` rule to have e.g. one run per php version, one run per database and one run per server version.